api/src/main/resources/application.properties

```diff
@@ -50,3 +50,7 @@ app.api.version=@project.version@
 app.cors.max_age_secs=3600
 spring.servlet.multipart.max-file-size=600MB
 spring.servlet.multipart.max-request-size =600MB
+spring.jpa.properties.hibernate.generate_statistics=true
+spring.jpa.properties.hibernate.jdbc.batch_size=200
+spring.jpa.properties.hibernate.order_inserts=true
+hibernate.cache.use_query_cache=true
```

Comment:
What are these changes related to?

Reply:
yes this will increase speed at upload and not failed on the edge env